voila/handler.py

```diff
@@ -69,7 +69,9 @@ def get(self, path=None):
         }
 
         # include potential extra resources
-        extra_resources = self.voila_configuration.resources
+        extra_resources = self.voila_configuration.config.VoilaConfiguration.resources
+        if not isinstance(extra_resources, dict):
+            extra_resources = extra_resources.to_dict()
```

Comment:
I don't understand this change.

Reply:
Wait, I should be more specific in my answer: when `PREFIX/share/jupyter/voila/templates/reveal/conf.json` does not exist *and no resource gets passed at the CLI*, `extra_resources` is a `traitlets.config.loader.LazyConfigValue` object.

Now, you may ask, but it's only a `traitlets.config.loader.LazyConfigValue` object because of the change you made right above, line 72! And that's correct, because `self.voila_configuration.resources` is always a dict. The reason I'm going for `self.voila_configuration.config.VoilaConfiguration.resources` is to support the case *with* config file, because `self.voila_configuration.resources` contains only resources passed at the CLI, not those in the config file. This in turns stems from the very specific config merge we are doing here: https://github.com/voila-dashboards/voila/pull/372/files#diff-024301d95616622c19f49b8cd78d33c3R372